docs/tutorial/maps.rst

```diff
@@ -181,748 +166,386 @@ Some basic statistical functions are built into Map objects:
-To be consistent with Matplotlib, sunpy has developed a standard plotting interface which supports both simple and advanced Matplotlib usage.
+This tells us that the coordinate system of the image is Helioprojective (HPC) and that it is defined by an observer at a particular location.
+This observer coordinate is also provided as an attribute,
 
-peek()
-------
-For quick and easy access to a plot `~sunpy.map.GenericMap` and `~sunpy.timeseries.GenericTimeSeries` (see next section), both define their own ``peek()`` methods which create a plot for you and show it without you having to deal with any Matplotlib setup.
-This is so that it is easy to take a quick look at your data.
+.. code-block:: python
 
-For example, to create a plot just type:
+    >>> my_map.observer_coordinate  # doctest: +REMOTE_DATA
+    <SkyCoord (HeliographicStonyhurst: obstime=2011-06-07T06:33:02.770, rsun=696000.0 km): (lon, lat, radius) in (deg, deg, m)
+        (-0.00406308, 0.04787238, 1.51846026e+11)>
 
-.. plot::
-    :include-source:
+This tells us the location of the spacecraft, in this case SDO, when it recorded this partiuclar observation, as derived from the FITS metadata.
+Map has several additional coordinate-related attributes that provide the coordinates of the center and corners of the Map,
 
-    import sunpy.map
-    import sunpy.data.sample
-    aia_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
-    aia_map.peek()
+.. code-block:: python
 
+    >>> my_map.center  # doctest: +REMOTE_DATA
+    <SkyCoord (Helioprojective: obstime=2011-06-07T06:33:02.770, rsun=696000.0 km, observer=<HeliographicStonyhurst Coordinate (obstime=2011-06-07T06:33:02.770, rsun=696000.0 km): (lon, lat, radius) in (deg, deg, m)
+        (-0.00406308, 0.04787238, 1.51846026e+11)>): (Tx, Ty) in arcsec
+        (3.22309951, 1.38578135)>
+    >>> my_map.bottom_left_coord  # doctest: +REMOTE_DATA
+    <SkyCoord (Helioprojective: obstime=2011-06-07T06:33:02.770, rsun=696000.0 km, observer=<HeliographicStonyhurst Coordinate (obstime=2011-06-07T06:33:02.770, rsun=696000.0 km): (lon, lat, radius) in (deg, deg, m)
+        (-0.00406308, 0.04787238, 1.51846026e+11)>): (Tx, Ty) in arcsec
+        (-1228.76466158, -1224.62447509)>
+    >>> my_map.top_right_coord  # doctest: +REMOTE_DATA
+    <SkyCoord (Helioprojective: obstime=2011-06-07T06:33:02.770, rsun=696000.0 km, observer=<HeliographicStonyhurst Coordinate (obstime=2011-06-07T06:33:02.770, rsun=696000.0 km): (lon, lat, radius) in (deg, deg, m)
+        (-0.00406308, 0.04787238, 1.51846026e+11)>): (Tx, Ty) in arcsec
+        (1235.21095899, 1227.39598836)>
 
-This creates a plot window with all axes defined, a plot title, and the image of the map data defined by the contents of the map.
-In non-interactive mode the plot window blocks execution and must be closed before doing anything else.
+But what if we wanted to know what pixel these physical coordinates correspond to?
```

Comment:
```suggestion
But what if we wanted to know what pixel these physical coordinates correspond to, or what physical coordinate corresponds to a particular pixel?
```

Reply:
Hmm to me, the suggested addition makes this sentence more confusing

Reply:
what about 
```suggestion
But what if we wanted to know what pixel these physical coordinates correspond to? or visa versa
```

Reply:
Sorry to keep pushing back on this, but I think the original wording works best in the flow of this section. We demonstrate map has these coordinate attributes and then the next natural question is what is the corresponding pixel coordinate? I think presenting things incrementally with limited scope is how a tutorial should be structured. Plus, the converse operation (pixel-to-world) is shown a few lines later so I don't think it would be possible for a user to leave this tutorial thinking WCS does not allow you to go both ways.